_posts/2016-01-22-pso.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: Pipeline State Objects
+---
+
+## Introduction
+
+PSO appears to be an integral part of the next-gen graphics APIs: DX12 [has them](https://msdn.microsoft.com/en-us/library/windows/desktop/dn899196%28v=vs.85%29.aspx), and [so does](https://developer.apple.com/library/ios/documentation/Metal/Reference/MTLRenderPipelineState_Ref/index.html) Metal. At Gfx-rs, we couldn't ignore this concept, and we went much further than simply adding support for it. We ditched our old abstractions (based around `Batch` concept) and put PSO at the center of the interface. This move has cleaned up Gfx-rs and brought it closer to the bare API level. The main challenge was making the PSO management type-safe and convenient without any run-time overhead.
```

Comment:
I'd expand the first reference from PSO to "Pipeline state objects (PSO) appear to be..."